gulpfile.js

```diff
@@ -3,12 +3,26 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+var path = require('path');
 var gulp = require('gulp');
 var clean = require('gulp-clean');
 var run = require('gulp-run')
 var mocha = require('gulp-mocha');
 var { restore, build, test, pack, publish } = require('gulp-dotnet-cli');
 
+// add .bin to PATH
+function getPathVariableName() {
+    // windows calls it's path 'Path' usually, but this is not guaranteed.
+    if (process.platform === 'win32') {
```

Comment:
Can you explain what's happening here?

Reply:
this function gets the PATH variable for the respective OS.
While it's (seemingly) always `PATH` on Linux, on Windows the capitalization can differ!
So we search for any variable that matches "path" (case insensitively) or fall back to `Path` which seems to be the usual casing in Win10